src/PrestaShopBundle/Resources/config/services/adapter/product/_image.yml

```diff
@@ -71,7 +71,19 @@ services:
         class: PrestaShop\PrestaShop\Adapter\Product\Image\ProductImagePathFactory
         arguments:
             - "@=service('prestashop.adapter.legacy.configuration').getBoolean('PS_LEGACY_IMAGES')"
+            - ''
```

Comment:
I wonder if we shouldn't rename `prestashop.adapter.product.image.product_image_path_factory` to `prestashop.adapter.product.image.product_image_file_path_factory` or even `prestashop.adapter.product.image.product_image_filesystem_path_factory`

To emphasize the difference with the URL factory and avoid any confusion

Reply:
idk i think path & url are fine. Looks good to me :man_shrugging: